--- /srv/bbx/baobaxia/app/django-bbx/bbx/settings.py	2014-12-17 19:08:33.144358938 -0200
+++ /srv/bbx/baobaxia/app/django-bbx/bbx/settings.py	2012-12-17 19:08:24.500359282 -0200
@@ -20,6 +20,14 @@
 MOCAMBOLA_DIR = "mocambolas"  # Nome da pasta onde estao os usuarios em .json
 DEFAULT_MUCUA = "dandara"
 DEFAULT_REPOSITORY = "mocambos"
+LOCALE_PATHS = (os.path.join(PROJECT_ROOT, 'locale'),)
+ugettext = lambda s: s
+LANGUAGES = [
+    ('en_US', ugettext('English')),
+    ('es_ES', ugettext('Spanish')),
+    ('pt_BR', ugettext('Brazilian Portuguese')),
+]
+USE_I18N = True
 
 POLICIES_DIR = os.path.join(PROJECT_ROOT, "policy")
 
@@ -133,6 +141,7 @@
 MIDDLEWARE_CLASSES = (
     'django.middleware.common.CommonMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.locale.LocaleMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
@@ -172,6 +181,7 @@
     'tag',
     'repository',
     'mocambola',
+    'lang',
     'south',
     'sorl.thumbnail',
 )
